interface.go

```diff
@@ -19,100 +19,46 @@ package reuseport
 
 import (
 	"context"
-	"errors"
 	"net"
-	"syscall"
-	"time"
+
+	"github.com/pkg/errors"
 )
 
-// Available returns whether or not SO_REUSEPORT is available in the OS.
-// It does so by attepting to open a tcp listener, setting the option, and
-// checking ENOPROTOOPT on error. After checking, the decision is cached
-// for the rest of the process run.
+// Available returns whether or not SO_REUSEPORT or equivalent behaviour is
+// available in the OS.
 func Available() bool {
-	return available()
+	return true
```

Comment:
I'm pretty sure we can't simply assume that reuseport is available.

Reply:
Got it. I'm fine dropping support for linux <= 3.9.

However, let's keep `Available` as we may need to (eventually) support platforms without reuseport support. Given that we already have it and it adds little complexity, we might as well keep it (unless we find the additional complexity a burden).